orttraining/orttraining/training_api/include/onnxruntime_training_c_api.h

```diff
@@ -4,6 +4,7 @@
 // This file contains the training c apis.
 
 #pragma once
+#include <stdbool.h>
```

Comment:
is this needed?

Reply:
Yeah, for `bool` support in C11 otherwise the type name is `_Bool` - https://en.cppreference.com/w/c/types/boolean